modules/building/pages/activation-keys-subscription.adoc

```diff
@@ -45,6 +52,82 @@ Alternatively, you can create the secret through the CLI. After logging into you
 kubectl create secret generic activation-key -n <your-tenant> --from-literal=org=<your org id> --from-literal=activationkey=<your activation key name>
 ----
 
+[[Create-custom-activation-key-secret]]
```

Comment:
Would it make sense to just combine with with the section above, `=== Create activation key secret through console`?

Reply:
Sounds good. I guess the other solution is much simpler in that you don't need to interact with creating CRs directly.